playwright/mockData/orgs/KPD/journeys/MarxistTraining/instances.ts

```diff
@@ -26,22 +27,50 @@ const milestones = [
 const groupIds = ids.splice(0, 3);
 const groupTags: ZetkinJourneyInstance['tags'][] = [
   [
-    { color: 'salmon', title: '1 - immediate' },
-    { color: 'peachpuff', title: '2 - near future' },
+    {
+      color: 'salmon',
```

Comment:
I don't really mind this in the mock data, but a value like this would never come from the API, which only supports colors in the format `#ffcc00` (six digits, preceded by '#').